src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+    private Random random = new Random();
```

Comment:
Avoid using an instance of the 'Random' class as a class member. It's better to create it inside the method where it's used if it's not used elsewhere.